nmosauth/auth_server/models.py

```diff
@@ -73,10 +73,10 @@ class OAuth2Token(db.Model, OAuth2TokenMixin):
     user_id = db.Column(
         db.Integer, db.ForeignKey('admin_user.id', ondelete='CASCADE'))
     admin_user = db.relationship('AdminUser')
-    access_token = db.Column(db.String(255), nullable=False, unique=False)
+    access_token = db.Column(db.String(255), nullable=False)
 
     def is_refresh_token_expired(self):
-        expires_at = self.issued_at + self.expires_in * 2
+        expires_at = self.issued_at + self.expires_in * 1440
```

Comment:
Not sure I understand the two changes in this file. Could you explain?

Reply:
1440 = number of minutes in a day - acts as a multiplier of the access token's expiry time